website/src/main/tut/docs/users/installation.md

```diff
@@ -25,10 +25,12 @@ addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "{{ site.stableVersion }}")
 > scalafix RemoveUnusedImports           // Run only RemoveUnusedImports rule
 > myProject/scalafix RemoveUnusedImports // Run rule in one project only
 > test:scalafix RemoveUnusedImports      // Run rule in single configuration
-> scalafix ExplicitR<TAB>                // use tab completion
-> scalafix replace:com.foobar/com.buzbaz // refactor (experimental)
-> scalafix file:rules/MyRule.scala       // run local custom rule
-> scalafix github:org/repo/v1            // run library migration rule
+> scalafix ExplicitR<TAB>                // Use tab completion
+> scalafix replace:com.foobar/com.buzbaz // Refactor (experimental)
+> scalafix file:rules/MyRule.scala       // Run local custom rule
+> scalafix github:org/repo/v1            // Run library migration rule
+> scalafixTest                           // Make sure no files needs to be fixed
+> scalafixCli <TAB>                      // Powerful autocompletion based on the cli options
```

Comment:
>  Invoke scalafix command line interface directly

Autocompletion should not be the main feature of this command, no matter how impressive it is 😜 

I think we should add a couple scalafixCli examples.

Reply:
Naw this is better with a screen capture, we can add examples if we have a shiny marketing website. Also, I don't want to copy paste the cli options below.

Reply:
This is currently the best we got :/ The main functionality of `scalafixCli` is running the cli, not the auto-completion. `scalafixCli --help` and `scalafixCli --diff ...` would be good examples. 

> I don't want to copy paste the cli options below.

Please do it, or remove the table. An incomplete table means users might think scalafixCli does not exist.

Reply:
It's still not clear what you want.
The list of arguments for scalafixCli task is listed below the section https://scalacenter.github.io/scalafix/docs/users/installation#help

Do you want me to add the missing task/settings?
https://scalacenter.github.io/scalafix/docs/users/installation#settings-and-tasks